README.md

```diff
@@ -1,42 +1,91 @@
 # POA ballot stats
 
-A command line tool that displays voting statistics for the [POA network](https://poa.network/).
-It and needs to communicate with a fully synchronized node that is connected to the network:
-[POA installation](https://github.com/poanetwork/wiki/wiki/POA-Installation).
-Note that `poa-ballot-stats` needs access to the network's full logs, so the node must run with
-`--pruning=archive --no-warp`. 
-Initial requirements for the tool described in RFC9 "Statistics of ballots." https://github.com/poanetwork/RFC/issues/9
+![logo](poa-logo.png)
 
-# Usage
+POA ballot stats is a command line tool used to display voting statistics for the [POA network](https://poa.network/). 
 
-## Stable release
+[Validators](https://github.com/poanetwork/wiki/wiki/Role-of-Validator) on the network engage in active governance, managing their roles and creating on-chain consensus. This is achieved through a balloting process. 
 
-Download the archive for your platform from the latest [release](https://github.com/poanetwork/poa-ballot-stats/releases) and unpack it. Run the tool with `./poa-ballot-stats <options>`.
+Ballot tracking provides transparency for POA token holders and promotes validator accountability. The poa-ballot-stats tool displays active validator voting participation ordered by percentage of missed votes. 
 
-You can view the command line options with `-h`, and specify a different endpoint if your node e.g.
-uses a non-standard port. By default, it tries to connect to a local node `http://127.0.0.1:8545`.
+The default display includes:
+* non-participation/associated ballots
+* missed %
+* truncated voting key
+* truncated mining key
+* first name last name
 
-In verbose mode, with `-v`, the list of collected ballot and key change events is displayed, and for each ballot the list of participating and abstaining voters.
+![Screenshot](screenshot.png)
 
-The `-c` option takes a map with the POA contracts' addresses in JSON format. You can find the 
-current maps for the main and test network the `contracts` folder. By default, it uses `core.json`,
-for the main network.
 
-The `-p` option takes a time interval in hours, days, months, etc. E.g. `-p "10 weeks"` will only count participation in ballots that were created within the last 10 weeks. Alternatively, instead of a _time_, you can specify the earliest block _number_ as a decimal integer with the `-b` option.
+## Dependencies
+Prior to downloading poa-ballot-stats, install and activate a fully synchronized node connected to the POA network. See the [POA installation guide](https://github.com/poanetwork/wiki/wiki/POA-Installation) for instructions.
 
-Examples:
+**Note:** poa-ballot-stats must access the full network logs. Use these flags when running a node `--pruning=archive --no-warp`
+
+**Example:**
+```bash
+$ parity --chain c:\path\to\spec.json --reserved-peers c:\path\to\bootnodes.txt --pruning=archive --no-warp
```

Comment:
Below we use Unix-style filenames (with `/` instead of `\`), and we currently don't provide Windows binaries at all; maybe the example files should be something like `poa-chain-spec/spec.json` and `poa-chain-spec/bootnodes.txt`.